templateflow/conf/__init__.py

```diff
@@ -76,6 +75,11 @@ def update_home(force=False):
     return False
 
 
-TF_LAYOUT = Layout(
-    str(TF_HOME), validate=False, config='templateflow',
-    ignore=['.git', '.datalad', '.gitannex', '.gitattributes', 'scripts'])
+TF_LAYOUT = None
+try:
+    from .bids import Layout
+    TF_LAYOUT = Layout(
+        str(TF_HOME), validate=False, config='templateflow',
```

Comment:
You should be able to drop the `str` now. That was a fix while we waited on bids-standard/pybids#457.
```suggestion
        TF_HOME, validate=False, config='templateflow',
```